packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.d.ts

```diff
@@ -1,16 +1,18 @@
-import { FunctionComponent, HTMLProps, MouseEvent, ReactNode, HTMLDivElement } from 'react';
-import { OneOf } from '../../typeUtils';
+import { FunctionComponent, HTMLProps, MouseEvent, ReactNode } from 'react';
+import { OneOf } from '../../helpers/typeUtils';
 import { PopoverPosition } from '../Popover';
 
-export const ClipboardCopyVariant = {
-  inline: 'inline',
-  expansion: 'expansion'
-};
+export enum ClipboardCopyVariant {
+  inline = 'inline',
+  expansion = 'expansion'
+}
 
 export interface ClipboardCopyProps extends HTMLProps<HTMLDivElement> {
   className?: string;
   hoverTip?: string;
   clickTip?: string;
+  'aria-label'?: string;
+  'toggle-aria-label'?: string;
   isReadOnly?: boolean;
   variant?: OneOf<typeof ClipboardCopyVariant, keyof typeof ClipboardCopyVariant>;
```

Comment:
seeing a couple more creep in here now in my IDE... i would be up for:

```
import { ClipboardEvent, FormEvent, FunctionComponent, HTMLProps, MouseEvent, ReactNode } from 'react';

export interface ClipboardCopyProps extends HTMLProps<HTMLDivElement> {
   ...
  onCopy?: (event: ClipboardEvent<HTMLDivElement>, text?: string) => void;
  onChange?: (event: FormEvent<HTMLDivElement>) => void;
```

or something else similar...

Reply:
or just remove them for now 😆